packages/atlas/src/views/global/YppLandingView/YppAuthorizationModal/YppAuthorizationSteps/YppAuthorizationDetailsFormStep/YppAuthorizationDetailsFormStep.tsx

```diff
@@ -159,7 +165,7 @@ export const YppAuthorizationDetailsFormStep: FC = () => {
             )
           }
           {...titleRegisterRest}
-          placeholder="Channel handle"
+          placeholder="Channel Name"
```

Comment:
I cannot comment on this line, but on line 156  we have `Enter the title of the Joystream channel which recommended the program to you.` This doesn't seem right to me as well. It would make sense to have `Joystream member` here, but not a channel. It should be `Enter the title of the ${atlasConfig.general.appName} channel which recommended the program to you.`

Could you also update this?